File: back-end/carbon-cruncher-api/Models/VisuInfo.cs
using System;
using System.Collections.Generic;

namespace carbon_cruncher_api.Models;

public partial class VisuInfo
{
    public int Id { get; set; }

    public int VisuNumber { get; set; }

    public int VisuChartNumber { get; set; }

    public string? Title { get; set; }

    public string? DescriptionText { get; set; }

    public string? DescriptionLink { get; set; }

    public string? DataLink { get; set; }
}

File: front-end/carbon-cruncher/src/components/VisuInfo/VisuInfo.js
import React from "react"
import styles from "./VisuInfo.module.css"

export const VisuInfo = ({ info }) => {
  return (
    <div className={styles.container}>
      <h1 className={styles.title}>{info.title}</h1>
      {!info.descriptionText ? <></> : <p>{info.descriptionText}</p>}
      <div className={styles.links}>
        {!info.dataLink ? (
          <></>
        ) : (
          <a href={info.dataLink} target="_blank" rel="noreferrer">
            Link to Data
          </a>
        )}
        {!info.descriptionLink ? (
          <></>
        ) : (
          <a href={info.descriptionLink} target="_blank" rel="noreferrer">
            Link to Description
          </a>
        )}
      </div>
    </div>
  )
}
